data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -636,6 +659,32 @@ class ReversePercentile:
 
         return df_copy
 
+    @staticmethod
+    def _get_island_areas(df: pd.DataFrame) -> pd.Series:
+        return (
+            df[field_names.GEOID_TRACT_FIELD]
+            .str[:2]
+            .isin(constants.TILES_ISLAND_AREA_FIPS_CODES)
+        )
+
+    def _backfill_island_demographics(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
is it worth adding a test for this? it seems like it might be straightforward to test and would add a lot of confidence.

Reply:
Good call --- done in 403c55a0ec8cc423963506c881a63a806379038a

It is an ugly test, but it is a test.

Reply:
Got it. I was thinking a test directly of `_backfill_island_demographics` (just give it a simple input DF and test the output DF), but your test works!